📝(content) Shree Bose Interview for IWD (#343)
chromeos/chromeos.dev: site/en/posts/iwd-2022-shree-bose-piper.md

---

> title: 'Helping students fall in love with building technology - Shree Bose of Piper'

Reviewer: ```suggestion
title: 'Helping students fall in love with building technology with Piper's Shree Bose'
```

Just a slight rewording to align with how Julia's article's title was written

Author: Changed!

metadesc: In honor of International Women's Day we interviewed Shree Bose, co-founder of Piper, about her quest to help people become passionate about creating technology.
tags:

>   - success story
>   - web

Reviewer: ```suggestion
    - web
    - success story
```
Because this is going to fall under our "web" blog category we want web to come first

authors:
  - hadrosaur
date: 2022-03-15
---

> ![Photo of Shree Bose wearing scrubs and a stethoscope, setting an infant into a crib in a hospital setting.](ix://posts/iwd-2022-shree-bose-piper/shree.jpeg)

Reviewer: This photo was previously uploaded to this URL and unfortunately there isn't an easy way to rename folders, so if you could update it to this that'd be great!

> ![Photo of Shree Bose wearing scrubs and a stethoscope, setting an infant into a crib in a hospital setting.](ix://posts/iwd-2022-shree-bose-piper/shree.jpeg)

Author: Fixed!

> As you are in that moment, feeling that you are sucking at something, think about different ways to teach yourself the same thing because if you don't give up, you might come up with something really, really amazing. And that's the most powerful thing you can do.

---

I am so grateful to have had the opportunity to talk with Shree and I cannot wait to try out some of the upcoming kits myself. Check out [Piper and Piper Make](https://www.playpiper.in/) today with your Chromebook!

> 
> Emilie Roberts, Developer Relations Engineer, Google Chrome OS

Reviewer: ```suggestion
```

You're already credited as the author through our template, so you don't need to include a by-line at the end.

Author: I am ok to not be credited here BUT I wonder if having doesn't make it clear that the remaining text is me and not Shree speaking?
